src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1318 @@
+            $aSections['disease']);
+        if (!empty($aDisease['name'])) {
+            list($aDisease['id'], $bCreateNewDisease) = lovd_getDiseaseID($aDisease['name']);
+            if ($bCreateNewDisease && $aDisease['id'] !== false) {
+                // New disease, create an output record for it.
+                $aDiseases[] = $aDisease;
+            }
+        }
+
+        // Handle individual-specific data (individual, screening, phenotype, etc.).
+        $sIndividualInputHeader = 'ID_patientid_';
+        if (($i = array_search($sIndividualInputHeader, $aInputHeaders)) !== false) {
+            $sLOVD2IndividualID = $aInputRecord[$i];
+            if (!isset($aIndividuals[$sLOVD2IndividualID])) {
+                // New individual, create an output record for it.
+                $aIndividual = lovd_getRecordForHeaders($aOutputHeaders['individual'], $aRecord,
+                    $aSections['individual']);
+                if ($aIndividual['created_by'] === false) {
+                    // No curator ID was available, set submitter ID.
+                    $aIndividual['created_by'] = $sSubmitterID;
+                }
+                if ($aIndividual['edited_by'] === false) {
+                    // No curator ID was available, set submitter ID.
+                    $aIndividual['edited_by'] = $sSubmitterID;
+                }
+                $aIndividual['owned_by'] = $sSubmitterID;
+                if (($nStatusIdx = array_search('ID_status_', $aInputHeaders)) !== false) {
+                    $aIndividual['statusid'] = $aRecord[$nStatusIdx];
+                }
+                $aIndividuals[$sLOVD2IndividualID] = $aIndividual;
+
+                // Create screening record.
+                $aScreening = lovd_getRecordForHeaders($aOutputHeaders['screening'], $aRecord,
+                    $aSections['screening']);
+                $nScreeningID = lovd_autoIncScreeningID();
+                $aScreening['id'] = $nScreeningID;
+                $aScreening['individualid'] = $aIndividual['id'];
+                $aScreening['owned_by'] = $aIndividual['owned_by'];
+                $aScreening['created_by'] = $aIndividual['created_by'];
+                $aScreening['created_date'] = $aIndividual['created_date'];
+                $aScreening['edited_by'] = $aIndividual['edited_by'];
+                $aScreening['edited_date'] = $aIndividual['edited_date'];
+                $aScreenings[$sLOVD2IndividualID] = $aScreening;
+
+                // Create screening2gene record.
+                $aScreening2Genes[] = array('screeningid' => $nScreeningID,
+                                            'geneid' => $zTranscript['geneid']);
+
+                // Create phenotype record.
+                $aPhenotype = lovd_getRecordForHeaders($aOutputHeaders['phenotype'], $aRecord,
+                    $aSections['phenotype']);
+                // Skip phenotype because there is no data in phenotype
+                // record except for ID fields.
```

Comment:
> What is this comment referring to?

Something either moved or not implemented yet, I suspect. I've implemented this in 1b458f5